scripts/jenkins/cloud/ansible/deploy-caasp-v4-terraform.yml

```diff
@@ -0,0 +1,565 @@
+      - name: caasp40-update
+        repo: http://ibs-mirror.prv.suse.net/ibs/SUSE:/SLE-15-SP1:/Update:/Products:/CASP40:/Update/standard/
+    master_size: "m1.medium"
+    network_tf_path: "{{ caasp_openstack_path }}/network.tf"
+    #Due to resource constraints in virtual cloud environment setup, the num_masters and num_workers are set to 1
+    #This can be overidden for baremetal QE cloud environments.
+    num_masters: 1
+    num_workers: 1
+    registration_auto_vars_path: "{{ caasp_openstack_path }}/registration.auto.tfvars"
+    repositories: '\n
+      basesystem_pool = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Basesystem/15-SP1/x86_64/product/" \n
+      serverapps_pool = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Server-Applications/15-SP1/x86_64/product/" \n
+      containers_pool = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Products/SLE-Module-Containers/15-SP1/x86_64/product/" \n
+      basesystem_updates = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Basesystem/15-SP1/x86_64/update/" \n
+      containers_updates = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Containers/15-SP1/x86_64/update/" \n
+      serverapps_updates = "http://ibs-mirror.prv.suse.net/ibs/SUSE/Updates/SLE-Module-Server-Applications/15-SP1/x86_64/update/" \n
+    '
+    ssh_key_filename: "id_rsa"
+    stack_name: "auto-caasp"
+    subnet_cidr: "172.28.0.0/24"
+    terraform_destroy: false
+    tfvars_path: "{{ caasp_openstack_path }}/terraform.tfvars"
+    timestamp: ""
+    service_caasp_osrc: "service_caasp.osrc"
+    user: "sles"
+    worker_size: "m1.medium"
+  tasks:
+  - name: Use python3
+    set_fact:
+      ansible_python_interpreter: "/usr/bin/python3"
+
+  - name: Management Machine | Get timestamp
+    set_fact:
+      timestamp: "{{lookup('pipe', 'date +%Y%m%d%H%M%S')}}"
+
+  - name: Managment Machine | Resources names with timestamp
+    # resources named with timestamp
+    set_fact:
+      stack_name: "{{stack_name + '-' + timestamp}}"
+      internal_net: "{{internal_net + '-' + timestamp}}"
+      internal_subnet: "{{internal_subnet + '-' + timestamp}}"
+
+  - name: Management Machine | Update resolv.conf
+    shell:
+      echo "nameserver 10.84.2.20" | sudo tee -a /etc/resolv.conf
+    args:
+      executable: /bin/bash
+    become: yes
+
+  - name: Management Machine | Add zypper repos
+    zypper_repository:
+      name: "{{ item.name }}"
+      repo: "{{ item.repo }}"
+      runrefresh: yes
+    loop: "{{ management_machine_repos }}"
+    when: item.enabled | default(true)
+    become: yes
+
+  - name: Managment Machine | Install SUSE-CaaSP-Management
+    zypper:
+      name: "SUSE-CaaSP-Management"
+      state: present
+      type: pattern
+    become: yes
+
+  - name: Management Machine | Copy caasp
+    shell: |
+      set -e
+      mkdir -p ~/caasp/deployment/
+      cp -r /usr/share/caasp/terraform/openstack/ ~/caasp/deployment/
```

Comment:
Why is there `caasp_openstack_path` variable, but here you are using `~/caasp/deployment/` directly?